rosetta/Dockerfile.pax

```diff
@@ -0,0 +1,56 @@
+# syntax=docker/dockerfile:1-labs
+ARG BASE_IMAGE=ghcr.io/nvidia/paxml:latest
+ARG GIT_USER_EMAIL=jax@nvidia.com
+ARG GIT_USER_NAME=NVIDIA
+
+FROM scratch as rosetta-source
+COPY . /
+
+FROM scratch as pax-mirror-source
+ADD --keep-git-dir=true https://github.com/google/paxml.git#main /
```

Comment:
There is an `--keep-git-dir` option now? That's amazing 😄

Reply:
I've seen this `bake` command, but to me, this looks like a simplified version of `build` that is meant for ease of use. Is there a particular feature that will help us in our workflow?